docs/overview/vision-of-the-thousand-brains-project.md

```diff
@@ -9,16 +9,18 @@ We call the implementation described herein "Monty", in reference to Vernon Moun
+There are numerous advantages to sensorimotor learning and reference frames. At a high level, you can think about all the ways humans are different from today's AI. We learn quickly and continuously, constantly updating our knowledge of the world as we go about our day. We do not have to undergo a lengthy and expensive training phase to learn something new. We interact with the world and manipulate tools and objects in sophisticated ways that leverage our knowledge of how things are structured. For example, we can explore a new app on our phone and quickly figure out what it does and how it works based on other apps we know. We actively test hypotheses to fill in the gaps in our knowledge. We also learn from multiple sensors and our different sensors work together seamlessly. For example, we may learn what a new tool looks like with a few glances and then immediately know how to grab and interact with the object via touch. Finally, the basis for decisions is considerably less opaque than that found in Large Language Models, etc.
 
 # Common Neural Algorithm
 
-One of the most important discoveries about the brain is that most of what we think of as intelligence, from seeing, to touching, to hearing, to conceptual thinking, to language, is created by a common neural algorithm. All aspects of intelligence are created by the same sensorimotor mechanism. In the neocortex, this mechanism is implemented in each of the thousands of cortical columns. This means we can create many different types of intelligent systems using a set of common building blocks. The architecture we are creating is built on this premise. Monty will provide the core components and developers will then be able to assemble widely varying AI and robotics applications using these components in different numbers and arrangements. Any engineer will be able to create AI applications using the Platform without requiring huge computational resources or background knowledge.
\ No newline at end of file
+One of the most important discoveries about the brain is that most of what we think of as intelligence, from seeing, to touching, to hearing, to conceptual thinking, to language, is created by a common neural algorithm. All aspects of intelligence are created by the same sensorimotor mechanism. In the neocortex, this mechanism is implemented in each of the thousands of cortical columns. This means we can create many different types of intelligent systems using a set of common building blocks. The architecture we are creating is built on this premise. Monty will provide the core components and developers will then be able to assemble widely varying AI and robotics applications using these components in different numbers and arrangements. Any engineer will be able to create AI applications using the Platform without requiring huge computational resources or background knowledge.
```

Comment:
I don't see any difference here and also diff checker doesn't show anything. Am I missing something? If not, can you remove this "change" from the PR?

Reply:
The last paragraph adds a new line to the end of the file. These don't show up in GitHub.

<img width="1134" alt="Screenshot 2024-12-12 at 09 36 02" src="https://github.com/user-attachments/assets/af31f374-7616-4604-809c-5a8e554ead33" />

Reply:
Ahh I see :D You would think github could show this as a bit of a smaller diff than the entire paragraph...